.github/workflows/ci.yml

```diff
@@ -178,3 +178,19 @@ jobs:
 
     - name: Test Dart
       run: cargo make test-dart
+  test-kotlin:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install cargo-make
+      uses: taiki-e/install-action@cargo-make
+    - name: Set up JDK 17 for x64
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
```

Comment:
As far as I can see @emarteca this now uses a JVM 17 version. I've also locally configured a temurin 17 JVM in my intelliJ and successfully ran the tests. So it should be good to go